tagstudio/src/core/library.py

```diff
@@ -343,6 +343,7 @@ def __init__(self) -> None:
         # A list of file extensions to be ignored by TagStudio.
         self.default_ext_blacklist: list = [".json", ".xmp", ".aae"]
         self.ignored_extensions: list = self.default_ext_blacklist
```

Comment:
I don't know if i should rename this variable and if i should i don't know what the new name should be.

Reply:
`default_ext_blacklist` should be fine since it's still true: it *is* the *default* extension *blacklist*.

`ignored_extensions` could be changed to something like "`ext_list`" since its purpose is now more generic, plus it brings the naming in line with `default_ext_blacklist`.

Reply:
Yep agree with that and have added it